86BoxManager/dlgSettings.cs

```diff
@@ -424,5 +273,27 @@ private void lnkGithub_LinkClicked(object sender, LinkLabelLinkClickedEventArgs
             lnkGithub.LinkVisited = true;
             Process.Start("https://github.com/86Box/86BoxManager");
         }
+
+        /// <summary>
+        /// Runs the exporter class code. 
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ExportSettingsFile_Click(object sender, EventArgs e)
+        {
+            SettingsExporter SEX = new SettingsExporter();
+            SEX.ZipUpRegFile = ExportZipButton.Checked;
+            
+            if (!SEX.ExportSettings())
```

Comment:
This code doesn't work out so well if the savefiledialog is canceled. What then?